tests/Harness.java

```diff
@@ -0,0 +1,77 @@
+/* vim: set filetype=java shiftwidth=4 tabstop=4 autoindent cindent expandtab : */
+
+import gnu.testlet.*;
+
+import com.sun.cldchi.jvm.JVM;
+import javax.microedition.lcdui.Display;
+import javax.microedition.lcdui.Form;
+import javax.microedition.midlet.*;
+import java.lang.Exception;
+import java.util.Vector;
+
+public class Harness extends TestHarness {
```

Comment:
Nit: it's strange that the name of this class, *Harness*, is more generic than the name of the superclass it extends, *TestHarness*. I would give it a more specific name than that class, like *RunTestsHarness* or *j2mejsTestHarness*.